programmers/난이도별/level01.문자열_내_p와_y의_개수/dkdlelk99.py

```diff
@@ -0,0 +1,11 @@
+def solution(s):
+    s = s.lower()
+    print(s)
+    p, y = s.count('p'), s.count('y')
+    if p == y:
+        return True
+    return False
```

Comment:
4 ~ 7 line은 그냥 이렇게 변경하는 게 좋을 것 같습니다.
```suggestion
    return s.count('p) == s.count('y')
```

Reply:
'가 빠진것 같아서 수정했습니다!
코드리뷰 감사합니다!